### PR TITLE
OFFLINE STOP, fix warn

### PR DESCRIPTION
**Pull request by BalaM314**
Fixes /warn, adds /stop_offline to stop an offline player (must be on the same server as the player was on)
Somewhat tested, but as the stopped api is ip whitelisted im not sure if it worked, but it did send the request

image
image

### CODE REVIEW

File: scripts/commands/staffCommands.js [hunk 1]
```diff
@@ -256,6 +256,59 @@ const registerCommands = (clientCommands, serverCommands, runner) => {
...
+      if (p.fakeAdmin) {
+        return;
+      }
+
+      if (!p.mod && !p.admin) {
+        realP.sendMessage('[scarlet]⚠ [yellow]You do not have access to this command.');
+        return;
+      }
+      let possiblePlayers = admins.searchNames(args[0]);
+      if(possiblePlayers.length > 20){
```
Comment (BalaM314) [inline scripts/commands/staffCommands.js]: This is wrong, ObjectSet does not have a length attribute

Comment (Brandons404) [reply scripts/commands/staffCommands.js]: Is this good to merge or no?

**Comment by Brandons404** [comment]
Thanks for doing this!

File: scripts/commands/staffCommands.js (hunk 1 above)
Comment (BalaM314) [reply scripts/commands/staffCommands.js]: It "works" but I should fix it

**Review by Brandons404 — APPROVED** [review]

**Pull request merged**